### PR TITLE
[web] Adjust expectation of dialog_test for Dart HHH bot

### PR DESCRIPTION
**Pull request by sigmundch**
dialog_test started failing in the Dart HHH bot since https://github.com/flutter/flutter/commit/c2409797ff19c81fed208706ccb760ca2b30f28e landed. When Chrome was rolled in flutter, the expectation of the test was updated, however, the Dart HHH still use an older version of Chrome (Chrome 84 I believe). Once that Chrome gets updated, we should revert this change.

/cc @nshahan

### CODE REVIEW

File: packages/flutter/test/cupertino/dialog_test.dart
Comment (yjbanov) [inline packages/flutter/test/cupertino/dialog_test.dart:336]: I think this will prevent the HHH bot from being able to upgrade to Chrome 96+ because as soon as it does so it will at the same time need a different expectation.

**Review by yjbanov — APPROVED** [review]
LGTM from Flutter's standpoint, but it may cause headaches in the future for the HHH bot maintainers.

File: packages/flutter/test/cupertino/dialog_test.dart
Comment (sigmundch) [reply packages/flutter/test/cupertino/dialog_test.dart:336]: I believe it will turn it red again, but it will not block from upgrading.  Given that this bot has been red for 1.5 months, I'd much rather get it back into a working state and manage the failures when the roll takes place by reverting this change at that time. @athomas @whesse - does that plan sound OK to you?

Two alternatives come to mind:
* Check for the Chrome version explicitly, instead. Is that available to us here? (not sure if this is worth it, though)
* Simply skip this line of the test in Dart HHH, so that the bot remains green after the roll.

Comment (nshahan) [reply packages/flutter/test/cupertino/dialog_test.dart:336]: This is fine with me but I do wonder if it would be easier to wait until the version of chrome on the HHH bot gets updated. @athomas @whesse It's not clear from our chat if that is going to be a difficult or time consuming task or not.

Comment (athomas) [reply packages/flutter/test/cupertino/dialog_test.dart:336]: Mainly I was waiting for the version to land in the SDK before updating 100, but the SDK CL showed some unexplained flakiness. I'll prepare the CL on Monday. Then you can decide when to land it.

Comment (athomas) [reply packages/flutter/test/cupertino/dialog_test.dart:336]: https://dart-review.googlesource.com/c/recipes/+/243502

Comment (sigmundch) [reply packages/flutter/test/cupertino/dialog_test.dart:336]: Thanks @athomas! I'll drop this PR then :).

**Pull request closed without merging**
